src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java

```diff
@@ -596,6 +596,18 @@ public static boolean isBlank(String value) {
                 || indexOfNonWhitespace(value) >= value.length();
     }
 
+    /**
+     * Checks if the value arg is blank by either being empty,
+     * or contains only whitespace characters.
+     *
+     * @param codePoints Array to check.
+     * @return true if the arg is blank.
+     */
+    public static boolean isBlank(int...codePoints) {
```

Comment:
Let's move this method and `indexOfNonWhitespace` to a new util class, `CodePointUtil`. Please move associated tests to a similarly named test class.

Reply:
done

Reply:
Resolved.